script.js

```diff
@@ -37,5 +37,6 @@ jQuery(() => {
     }
 
     jQuery('#wiki__text').hide();
+    jQuery('#size__ctl').hide();
```

Comment:
we will want the size controls back later on. I'll merge this but we should keep the ticket open.